source/Ship.cpp

```diff
@@ -1392,13 +1393,20 @@ vector<string> Ship::FlightCheck() const
 			checks.emplace_back("solar power?");
 		if(fuel < 0.)
 			checks.emplace_back("fuel?");
-		if(!canBeCarried)
+		if(canBeCarried && !intrasolar)
 		{
 			if(!hyperDrive && !jumpDrive)
 				checks.emplace_back("no hyperdrive?");
 			if(fuelCapacity < navigation.JumpFuel())
 				checks.emplace_back("no fuel?");
```

Comment:
```suggestion
```

I am thinking that it'd be best to just remove the fuel check from here, so that it can instead be a stand-alone check that triggers on the presence of a means of jumping instead of being based on whether or not it can be carried.